app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(params: Map[String, Any])
+@title = @{"Requested cash account transactions are now available"}
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, title) {
+
+    @uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.salutationLine1(params)
+
+    <div style="font-size: 19px;line-height: 1.315789474;margin: 0 0 30px 0;">
```

Comment:
Does it matter that we're not using components and its just raw HTML?

Reply:
not for email templates

Reply:
safe